Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Views/SplashView.cs

```diff
@@ -0,0 +1,19 @@
+using Android.App;
+using Android.Content.PM;
+using MvvmCross.Core.ViewModels;
+using MvvmCross.Droid.Views;
+using MvvmCross.Platform;
+
+namespace MvvmSeed.Android.Views
+{
+    [Activity(MainLauncher = true, NoHistory = true, ScreenOrientation = ScreenOrientation.Portrait, Theme = "@style/Theme.Splash")]
```

Comment:
Your phone will implode.. 

Or, the splash view will be shown in portrait mode regardless your phone orientation. Usually splash views are portrait-only.. if we want orientation-aware splash, we'll need a different asset for the background, and in that case we remove that attribute and adjust the theme accordingly.